resources/app.py

```diff
@@ -185,6 +201,14 @@ def get_room_by_id(room_id: int, db=Depends(get_db)):
     return room
 
 
+@app.get("/get/room/department/{department_id}", tags=["Room"])
+def get_room_by_department(department_id: int, db=Depends(get_db)):
```

Comment:
probably 'get_rooms_by_department' I know maybe I'm picking on but we get list of rooms not single one